subtitles.go

```diff
@@ -428,6 +428,94 @@ func (sa *StyleAttributes) propagateWebVTTAttributes() {
 	sa.SRTBold = sa.WebVTTBold
 	sa.SRTItalics = sa.WebVTTItalics
 	sa.SRTUnderline = sa.WebVTTUnderline
+
+	// may be overridden by position parsing later
+	switch sa.WebVTTAlign {
+	case "left", "right", "center", "start", "end":
+		sa.TTMLTextAlign = astikit.StrPtr(sa.WebVTTAlign)
+	}
+
+	for _, tag := range sa.WebVTTTags {
+		switch tag.Name {
+		case "c":
+			if len(tag.Classes) > 0 {
+				for _, color := range tag.Classes {
+					if strings.HasPrefix(color, "bg_") && len(color) > 3 {
+						if bgColor, err := parseWebVTTColorClass(color[3:]); err == nil {
+							sa.TTMLBackgroundColor = astikit.StrPtr("#" + bgColor.TTMLString())
+						}
+					} else {
+						if fgColor, err := parseWebVTTColorClass(color); err == nil {
+							sa.TTMLColor = astikit.StrPtr("#" + fgColor.TTMLString())
+						}
+					}
+				}
+			}
+		}
+	}
+
+	// Handle WebVTT position and alignment conversion
+	sa.handleWebVTTPositioning()
+
+	if sa.WebVTTSize != "" {
+		sa.TTMLExtent = astikit.StrPtr(fmt.Sprintf("%s 10%%", sa.WebVTTSize))
+	}
+}
+
+// parseWebVTTPosition parses WebVTT position string like "10%,line-left".
+// Returns (xPosition, alignment)
+func (sa *StyleAttributes) parseWebVTTPosition() (string, string) {
```

Comment:
Instead could you create a `WebVTTPosition` struct in `webvtt.go` with `XPosition` and `Alignment` attributes and make `StyleAttributes.WebVTTPosition` a `*WebVTTPosition`. Parsing would occur [here](https://github.com/asticode/go-astisub/blob/master/webvtt.go#L275) but rename the function to `newWebVTTPosition()`